src/dd-doctor.php

```diff
@@ -120,7 +120,8 @@ function env($key)
 function check_opcache()
 {
     if (!function_exists('opcache_get_configuration')) {
-        render('Opcache enabled:', 'NO');
+       render('Opcache enabled:', 'NO');
+       return;
```

Comment:
Oops! I messed up the indent.
```suggestion
        render('Opcache enabled:', 'NO');
        return;
```